style.css

```diff
@@ -0,0 +1,97 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  height: 92px;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-around;
```

Comment:
- [ ] Kindly use the property value of `space-between` to avoid the paddings around the header in order to match the Figma design
![Screen Shot 2022-05-26 at 05 40 10](https://user-images.githubusercontent.com/28307960/170411537-dd058b4f-69d4-4846-9a3d-d75950ad1da9.png)